ProcessEpoch.thy

```diff
@@ -422,37 +375,16 @@ definition initiate_validator_exit :: "u64 \<Rightarrow> (unit, 'a) cont"
        let exit_queue_epoch = maximum activation_exit_epoch exit_epochs;
        let exit_queue_churn = length (filter (\<lambda>v. exit_epoch_f v = exit_queue_epoch) (var_list_inner vs));
        churn_limit <- get_validator_churn_limit;
-       exit_queue_epoch \<leftarrow> (if (nat_to_u64 exit_queue_churn \<ge> churn_limit) then return exit_queue_epoch 
+       exit_queue_epoch \<leftarrow> (if (nat_to_u64 exit_queue_churn < churn_limit) then return exit_queue_epoch
                                                                  else exit_queue_epoch .+ Epoch 1);
        let val = val\<lparr>exit_epoch_f :=  exit_queue_epoch\<rparr>;
        new_withdrawable_epoch \<leftarrow> epoch_to_u64 (exit_epoch_f val) .+ MIN_VALIDATOR_WITHDRAWABILITY_DELAY config;
        let val = val\<lparr>withdrawable_epoch_f := Epoch (new_withdrawable_epoch) \<rparr>;
-       let vs = var_list_update val vs index;
-       return ()
+       vs \<leftarrow> var_list_update val vs index;
+       write_to validators vs
```

Comment:
I think we need to write the updated list of validators here, right? Otherwise the mutation will be lost